components/resources/ai_chat_ui_strings.grdp

```diff
@@ -264,12 +264,18 @@
   <message name="IDS_CHAT_UI_EDIT_ANSWER_LABEL" desc="Button title to edit AI agent answer">
     Edit answer
   </message>
-  <message name="IDS_CHAT_UI_REGENERATE_ANSWER_BUTTON_LABEL" desc="Button title to regenerate AI agent answer">
-    Regenerate
+  <message name="IDS_CHAT_UI_RETRY_SAME_MODEL_BUTTON_LABEL" desc="Button title to regenerate AI agent answer">
+    Retry same model
+  </message>
+  <message name="IDS_CHAT_UI_CURRENT_LABEL" desc="Label for current model">
+    Current
   </message>
   <message name="IDS_CHAT_UI_REGENERATE_ANSWER_MENU_TITLE" desc="Title of menu to regenerate AI agent answer">
     Change model for answer
   </message>
+  <message name="IDS_CHAT_UI_REGENERATE_ANSWER_MENU_TITLE_WITH_MODEL" desc="Title of menu to regenerate AI agent answer with a specific model">
+    Change model for answer (<ph name="MODEL_NAME">$1<ex>Qwen 14B</ex></ph>)
+  </message>
```

Comment:
The description says it's the menu title, but this appears to be a tooltip?
I think it's not clear what is in parentheses. Maybe specify `Change model for answer (current model: <ph name="MODEL_NAME">$1<ex>Qwen 14B</ex></ph>)`